ENTERPRISE-MIB

```diff
@@ -0,0 +1,151 @@
+CLEARWATER-ALARMS-MIB DEFINITIONS ::= BEGIN
```

Comment:
The normal convention is for the filename to match the MIB name - i.e. CLEARWATER-ALARMS-MIB not ENTERPRISE-MIB.